### PR TITLE
Added support for official kailh choc stabilisers

### PR DESCRIPTION
**Pull request by wejrox**
Kailh choc stabilisers have a footprint which does not match any of the current options available.
- Based on the specifications within the product listing on Aliexpress https://www.aliexpress.com/item/33039182740.html I have added it to the list of available stabiliser types.
- There are only two stabiliser sizes available; 2u and 6.25u.
- The official keycaps place the stabiliser holes slightly higher on the Y axis than the main switch socket, so that has also been accounted for.

### CODE REVIEW

**Comment by wejrox** [comment]
I've just received my cutouts with the new shape, and although they do fit the stabilisers, I made an oversight in that the height of the stabiliser means it should sit on TOP of the switchplate, with a hole for the dip in the stabiliser to sit in. This is because the height of the stabiliser matches the switch height when seated in the plate, leading to the stabiliser only sitting correctly if it sits next to the switch on the plate (as opposed to being pushed into a hole for the stabiliser in the plate itself. 

Basically, if you use this right now it'll be slightly too low when the keycap is attached, so the design will need to be updated slightly before merging.

**Comment by jvdp** [comment]
@wejrox I'm also interested in creating a custom keyboard with kailh chocs so this looks super interesting, thanks for sharing! Do you have any pointers on how I can use your branch to do that? Did you use http://builder.swillkb.com/ to configure it and then hand-translate it into Go code?

And do you have some specifics on the slight updates to the design? 😇

**Comment by wejrox** [comment]
@jvdp I forked this repository and modified the code to add a new stabiliser shape. I used the official specification for the dimensions of the stabiliser.  
To execute the repository I had to follow the steps in #4 because I was receiving the same errors.  
In order to test my changes I created a custom external GO test file which I executed in VSCode, and am using that to generate my cutouts.  
  
In essence, the stabilisers need to sit on top of the switch plate instead of being inserted into it. There needs to be a square cutout for the loose plastic part of the stabiliser (which connects to the keycap) to move up and down through the switchplate, and also a hole for the square shaped bit of plastic which protrudes out from the stabiliser (the bit that holds the stabiliser bar).  

The cutout that I created will still work well if you wanted to design a bit of cardboard for the stabiliser to attach to (since otherwise you'd need to glue it onto the switchplate directly). I'd recommend using a [cutout plate](https://www.reddit.com/r/olkb/comments/7hscl3/kailh_pg1350_2u_stabiliser_plate/?utm_source=share&utm_medium=web2x) to attach it to the board because it removes the risk of glue interfering with the stabiliser.  

I'm going to continue working on this branch and get it to a better state.
